bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -0,0 +1,248 @@
+      bus_automation           LR_shutter       "Living Room Shutter"     [ where="93", shutterRun="10050"]
+}
+``` 
+
+
+```xtend
+// ZigBee USB Gateway configuration: only needed for radio devices
+Bridge openwebnet:zb_gateway:myZBgateway  [serialPort="kkkkkkk"] {
+    zb_dimmer          myzigbeedimmer [ where="xxxxx"]
+    zb_on_off_switch   myzigbeeswitch [ where="yyyyy"]
+}
+```
+
+### openwebnet.items:
+
+Items (Light, Dimmer, etc.) will be discovered by Google Assistant/Alexa/HomeKit if their tags are configured like in the example.
+
+```xtend
+Switch			iLR_switch			"Light"								<light>          (gLivingRoom)                [ "Lighting" ]  { channel="openwebnet:bus_on_off_switch:mybridge:LR_switch:switch" }
+Dimmer			iLR_dimmer			"Dimmer [%.0f %%]"					<DimmableLight>  (gLivingRoom)                [ "Lighting" ]  { channel="openwebnet:bus_dimmer:mybridge:LR_dimmer:brightness" }
+Dimmer			iLR_dalidimmer		"Dali-Dimmer [%.0f %%]"				<DimmableLight>  (gLivingRoom)                [ "Lighting" ]  { channel="openwebnet:bus_dimmer:mybridge:LR_dalidimmer:brightness" }
+/* For Dimmers, use category DimmableLight to have Off/On switch in addition to the Percent slider in PaperUI */
+Rollershutter	iLR_shutter			"Shutter [%.0f %%]"					<rollershutter>  (gShutters, gLivingRoom)     [ "Blinds"   ]  { channel="openwebnet:bus_automation:mybridge:LR_shutter:shutter" }
+```
+
+### openwebnet.sitemap
+
+```xtend
+sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
+{
+    Frame label="Living Room"
+    {
+          Default item=iLR_switch           icon="light"    
+          Default item=iLR_dimmer           icon="light" 
+          Default item=iLR_dalidimmer       icon="light"
+          Default item=iLR_shutter
+    }
+}
+```
+
+## Disclaimer
+
+- This binding is not associated by any means with BTicino or Legrand companies
+- Contributors of this binding have no liability for any direct, indirect, incidental, special, exemplary, or consequential damage to things or people caused by using the binding connected to a real BTicino/Legrand (OpenWebNet) plant/system and its physical devices. The final user is the only responsible for using this binding in a real environment. See Articles 5. and 6. of [Eclipse Public Licence 2.0](https://www.eclipse.org/legal/epl-2.0/) under which this binding software is distributed
+- The OpenWebNet protocol is maintained and Copyright by BTicino/Legrand. The documentation of the protocol if freely accessible for developers on the [MyOpen Community website - https://www.myopen-legrandgroup.com/developers](https://www.myopen-legrandgroup.com/developers/)
+- OpenWebNet, MyHOME, MyHOME_Play and Living Now are registered trademarks by BTicino/Legrand
+- This binding uses `openwebnet-lib 0.9.x`, an OpenWebNet Java lib partly based on [openwebnet/rx-openwebnet](https://github.com/openwebnet/rx-openwebnet) client library by @niqdev, modified to support:
```

Comment:
If I read this correctly the library here is a modified version, but I don't see any links to the source of that modified library only to the original library?

Reply:
Hi @Hilbrand , thanks for starting the review. The library used started as a fork of [openwebnet/rx-openwebnet](https://github.com/openwebnet/rx-openwebnet), but then there were too many missing parts (mainly the USB serial part and events handling) so I started deviating quite a bit from that lib.
Current status is that I have a quite messy lib that I did not publish yet on GitHub.
It would take 4/5 weeks at least for me to clean up the library, refactor some parts to be decent, update the binding to use the new library and do some tests.
This would mean to miss the OH 2.5.0 release planned in December, and that would be not good at all since many users are waiting for this binding to be part of the official release.
I propose the following: the current binding is merged with the current lib, then after 2.5.0 release I will add more devices (WHOs) and the new restructured openwebnet lib to communicate with openwebnet gateways. What do you think?
In any case I want to keep the lib separated as it is quite useful to have a OpenWebNet Java library with all the characteristics I added: nothing similar can be found on opensource and other projects may benefit from such a lib, not just OH. Current dependencies of my openwebnet lib are: slf4j-log4j12, nrjavaserial, com.google.guava, org.eclipse.jdt.annotation

Reply:
Hi @Hilbrand, I think I addressed all your recommendations, correct?
If yes, would it be possible to continue your review of the binding?

Reply:
The binding is now based on openwebnet4j lib bundle, available here: https://github.com/mvalla/openwebnet4j